modules/hcp-virt-create-hc-console.adoc

```diff
@@ -22,7 +22,7 @@ To create a hosted cluster with the KubeVirt platform by using the console, comp
 +
 [NOTE]
 ====
-* If you want to use predefined values to automatically populate fields in the console, you can create a {VirtProductName} credential. For more information, see _Creating a credential for an on-premises environment_.
+* If you want to use predefined values to automatically populate fields in the console, you can create a {VirtProductName} credential. For more information, see _Creating a credential for an on-premise environment_.
```

Comment:
I think this should stay as "on-premises" because it's referencing a title that uses that term.